projects/plugins/jetpack/modules/theme-tools/site-logo/inc/class-site-logo.php

```diff
@@ -109,29 +106,49 @@ public function customize_register( $wp_customize ) {
 			'site_logo',
 			array(
 				'capability'        => 'manage_options',
-				'default'           => array(
-					'id'    => 0,
-					'sizes' => array(),
-					'url'   => false,
-				),
+				'default'           => 0,
 				'sanitize_callback' => array( $this, 'sanitize_logo_setting' ),
 				'transport'         => 'postMessage',
 				'type'              => 'option',
 			)
 		);
 
+		$size_settings = jetpack_get_site_logo_dimensions();
+		$width         = isset( $size_settings['width'] ) ? $size_settings['width'] : null;
+		$height        = isset( $size_settings['height'] ) ? $size_settings['height'] : null;
+
 		// Add our image uploader.
 		$wp_customize->add_control(
-			new Site_Logo_Image_Control(
+			new WP_Customize_Cropped_Image_Control(
 				$wp_customize,
 				'site_logo',
 				array(
-					'label'    => __( 'Logo', 'jetpack' ),
-					'section'  => 'title_tagline',
-					'settings' => 'site_logo',
+					'label'         => __( 'Logo', 'jetpack' ),
+					'section'       => 'title_tagline',
+					'settings'      => 'site_logo',
+					'width'         => $width,
+					'height'        => $height,
+					'button_labels' => array(
+						'select'       => __( 'Add logo', 'jetpack' ),
+						'change'       => __( 'Change logo', 'jetpack' ),
+						'remove'       => __( 'Remove logo', 'jetpack' ),
+						'placeholder'  => __( 'No logo set', 'jetpack' ),
+						'frame_title'  => __( 'Set as logo', 'jetpack' ),
+						'frame_button' => __( 'Choose logo', 'jetpack' ),
```

Comment:
This are the same translated strings from the custom control (that's now being deleted).